<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GMP</string>
  </property>
  <widget class="QPushButton" name="bt_init">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>INIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reboot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>REBOOT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_test">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_apply">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>APPLY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_observe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OBSERVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_abort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ABORT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_expTime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>174</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Exp.Time:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_acq">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ACQ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_sci">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SCI</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_ActionID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>16</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Action ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_send">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_recv">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>341</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>10</y>
     <width>20</width>
     <height>341</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>24</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_port">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_currentRMA">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>54</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>current RMA:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_positionAngle">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_currentDec">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_currentRA">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>114</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>current RA:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>144</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>current Dec:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>84</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>PA:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>174</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>airMass:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_airMass">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>204</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HA:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_HourAngle">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_localSiderealTime">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>234</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LST:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_zenithDistance">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>264</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ZD:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_send_fromGemini">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>293</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
